src/server.ts

```diff
@@ -13,6 +14,10 @@ app.get("/", (request, response) => {
   response.json({ message: "Server starts successfully!" });
 });
 
+// routes
+app.use("/api/v1", EventRoute);
```

Comment:
```suggestion
app.use("/api/v1/event", EventRoute);
```

Reply:
i dont think it will work since i have this in **`event-route.ts`**  `router.get("/event", EventsController.getEvents);` if i change that i will get a 404 when i test the route

Reply:
but doesn't this mean that all routes will be handled by the EventRoute? for example `/api/v1/opportunities` would still be handled by `EventRoute`? Is this right?

Reply:
```
app.use("/api/v1/event", EventRoute);
app.use("/api/v1/opportunities", OpportunityRoute);
```
@fatmali would this be heading in this direction?

Reply:
@MarvinKweyu ,just resolved that,have a look at my latest commit